### PR TITLE
Nullsafety & Flutter 2.5.3

### PR DESCRIPTION
**Pull request by praveen-gm**
## What?
I've updated the flutter version to <2.5.3, from the branch 'nullsafety'.

## Testing?
9 on 9 cases passed.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   CoreLocation for iOS.
-version: 0.3.0-nullsafety.0
+version: 0.3.1
 homepage: https://github.com/pszklarska/beacon_broadcast
 
 environment:
-  sdk: '>=2.12.0-29.10.beta <3.0.0'
-  flutter: ">=1.20.0 <2.0.0"
+  sdk: '>=2.12.0 <3.0.0'
+  flutter: ">=1.20.0 <2.5.3"
```
Comment (pszklarska) [inline pubspec.yaml:9]: There's no need to specify upper constraint, it's fine to use `flutter: ">=1.17.0"`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.3.1
+
+Null Safety
```
Comment (pszklarska) [inline CHANGELOG.md:3]: Can you elaborate that, e.g. change to "Added support for null safety"?

**Comment by pszklarska** [comment]
Thanks for that PR @praveen-gm, I added few comments

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 0.3.1
```
Comment (pszklarska) [inline CHANGELOG.md:1]: That should be 0.3.0

File: pubspec.yaml [hunk 4]
```diff
@@ -1,19 +1,19 @@
 name: beacon_broadcast
 description: A Flutter plugin for turning your device into a beacon. Plugin uses AltBeacon library for Android and
   CoreLocation for iOS.
-version: 0.2.3
+version: 0.3.1
```
Comment (pszklarska) [inline pubspec.yaml:4]: `0.3.0`

**Review by praveen-gm — COMMENTED** [review]
I've made the changes @pszklarska. Thanks for creating this package.

**Review by pszklarska — APPROVED** [review]

**Pull request merged**